tickets/src/main/java/no/unit/nva/publication/ticket/create/CreateTicketHandler.java

```diff
@@ -71,12 +76,29 @@ private static String createTicketLocation(SortableIdentifier publicationIdentif
                    .getUri()
                    .toString();
     }
-    
+
+    private static URI getPublisherId(Publication publication) {
+        return Optional.ofNullable(publication.getPublisher()).map(Organization::getId).orElse(null);
+    }
+
+    private boolean userIsAuthorized(RequestInfo requestInfo, Publication publication) throws UnauthorizedException {
+        return (requestInfo.userIsAuthorized(AccessRight.APPROVE_DOI_REQUEST.toString())
+                || requestInfo.userIsAuthorized(AccessRight.REJECT_DOI_REQUEST.toString()))
+               && isUserFromSameCustomerAsPublication(requestInfo, publication);
```

Comment:
jeg hadde kanskje forkortet denne noe, for eksempel:
```
return hasValidAccessRights(requestInfo) && matchingCustomer(requestInfo, publication);
```

Reply:
fikset